blocks/init/src/Blocks/custom/featured-categories/featured-categories.php

```diff
@@ -30,15 +30,18 @@
 	$args = [
 		'hide_empty' => false,
 		'taxonomy' => $taxonomy,
+		'orderby' => 'include',
+		'include' => array_map(
+			function ($item) {
+				return $item['value'];
+			},
+			$terms
+		),
```

Comment:
Kses should be removed from the components inside blocks, otherwise, you can have breakage (attributes missing etc.) On line 63 (couldn't comment there)

Reply:
fixed